src/main.c

```diff
@@ -103,6 +105,10 @@ static void registry_add(void* data, struct wl_registry* registry, uint32_t id,
 		xdg_wm_base = wl_registry_bind(registry, id, &xdg_wm_base_interface, 1);
 	} else if (strcmp(interface, "wl_shm") == 0) {
 		wl_shm = wl_registry_bind(registry, id, &wl_shm_interface, 1);
+	} else if (strcmp(interface, zwlr_input_inhibit_manager_v1_interface.name) == 0) {
+		struct zwlr_input_inhibit_manager_v1* manager = wl_registry_bind(registry, id, &zwlr_input_inhibit_manager_v1_interface, 1);
```

Comment:
The rest of the bindings here are assigned into global variables named after their respective interfaces. This should follow the same pattern.

This pattern breaks down if you want the wayland client to connect to multiple compositors, but that wouldn't really make any sense.